cmd/sensor-config/settings.go

```diff
@@ -109,7 +109,7 @@ func (s Settings) Groups(set *meta.Set) Network {
 		network.Groups = append(network.Groups, group)
 	}
 
-	if group, ok := s.Cameras(set, "Volcano camera", s.volcano); ok {
+	if group, ok := s.Cameras(set, "Camera", s.camera); ok {
 		network.Groups = append(network.Groups, group)
```

Comment:
are `s.volcano` and `s.camera` the same?

Reply:
no, at the moment the volcano one is used for the existing system (non-group) whereas camera is the new group version and is a list of networks (rather than a single network code).